tests/unit/_experimental/object_detection/test_configurations.py

```diff
@@ -80,7 +86,7 @@ def test__object_detection__multiple_configurations__multiclass() -> None:
 def test__object_detection__multiple_configurations__single_class() -> None:
     ground_truth = GroundTruth(bboxes=[LabeledBoundingBox(top_left=(0, 0), bottom_right=(1, 1), label="a")])
     inference = Inference(
-        bboxes=[ScoredLabeledBoundingBox(top_left=(0, 0), bottom_right=(1, 1), label="b", score=random.random())],
+        bboxes=[ScoredLabeledBoundingBox(top_left=(0, 0), bottom_right=(1, 1), label="a", score=random.random())],
```

Comment:
is this change needed?

Reply:
Not needed but the test was inaccurate. This isn't single class since there is `a` and `b`. I can remove it but figure it's overkill to have it's own pr

Reply:
The reason for the inaccuracy is to make sure that the single class evaluator doesn't care about class names

Reply:
ah got it. I'll revert this portion